ui/apps/platform/src/types/policy.proto.ts

```diff
@@ -100,10 +108,16 @@ export type PolicySection = {
     policyGroups: PolicyGroup[];
 };
 
-export type ValueObj = {
-    source?: string;
-    key?: string;
-    value?: string;
+type ClientPolicySection = {
+    sectionName: string;
+    policyGroups: ClientPolicyGroup[];
+};
+
+type ClientPolicyGroup = {
```

Comment:
It seems better to omit `fieldKey` property from `ClientPolicyGroup` type, unless stricter types in Step3?